src/commands/edge/drivers/package.ts

```diff
@@ -185,7 +198,10 @@ export default class PackageCommand extends EdgeCommand {
 			tableFieldDefinitions: ['driverId', 'name', 'packageKey', 'version'],
 		}
 
-		outputItem(this, config, async () => {
+		if (flags.upload) {
+			const data = fs.readFileSync(flags.upload)
```

Comment:
Optional, but may want to craft a friendlier message for the user than Node's `ENOENT` here when the file doesn't exist since that will be hit often with typos and such.

Reply:
I made this say `No file named "${flags.upload}" found`.